server/parsec/components/invite.py

```diff
@@ -459,7 +459,7 @@ def _event_filter(
                     # Hence it's hazardous to send the event directly from this coroutine (it
                     # requires cancellation shielding), and instead the `send_nowait` will
                     # delegate the work to a dedicated coroutine.
-                    status = InvitationStatusField.IDLE
+                    status = InvitationStatusField.IDLE  # pyright: ignore [reportAttributeAccessIssue]
```

Comment:
Why access issue, can you add a comment above ?

Reply:
I don't know why the access issue. It seems that pyright doesn't follow the `Annotated` type, but I couldn't find any reference nor related issue in their repo :confused:

Without the ignore, pyright reports:
```
error: Cannot access member "IDLE" for type "type[Annotated]"
```